custom/enikshay/integrations/bets/repeater_generators.py

```diff
@@ -28,6 +28,20 @@ class BETSPayload(jsonobject.JsonObject):
     BeneficiaryType = jsonobject.StringProperty(required=True)
     Location = jsonobject.StringProperty(required=True)
 
+    @classmethod
+    def _get_location(cls, location_id, field_name=None, related_case_type=None, related_case_id=None):
+        try:
+            return SQLLocation.objects.get(location_id=location_id)
+        except SQLLocation.DoesNotExist:
+            msg = "Location with id {location_id} not found.".format(location_id)
+            if field_name and related_case_type and related_case_id:
+                msg += " This is the {field_name} for {related_case_type} with id: {related_case_id}".format(
+                    field_name=field_name,
+                    related_case_type=related_case_type,
+                    related_case_id=related_case_id,
+                )
+        raise NikshayLocationNotFound(msg)
```

Comment:
looks like this `raise` is outside of the `except` block. Is that intended? if so, `msg` will be undefined at this point.